firebase-storage/src/main/java/com/google/firebase/storage/internal/Util.java

```diff
@@ -104,13 +101,7 @@ public static Uri normalize(@NonNull FirebaseApp app, @Nullable String s)
       if (scheme != null
           && (equals(scheme.toLowerCase(), "http") || equals(scheme.toLowerCase(), "https"))) {
         String lowerAuthority = uri.getAuthority().toLowerCase();
-        int indexOfAuth;
-        try {
```

Comment:
Not sure what this block did before but `RemoteException` is no longer thrown by anything in the path here so the compiler told me to drop this.

Reply:
`RemoteException` is dedicated to the long term memory of GMSCore. If you think we should not include references to these dark times, then we can remove it :)